lib/gamora/configuration.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Gamora
+  module Configuration
```

Comment:
Will this config file allow to have different openId servers?

Reply:
From the same issuer? I didn't get it.

Reply:
lets say I want to have a login with Amco IDP but also a GSuite IDP client, will this be posible with this implementation?

Reply:
or this only works for Amco IDP? (as the names says :))